### ISSUE TITLE
Gene:phenotype association display

### ISSUE THREAD

**Issue by mvelinder** [iobio/sv.iobio#124]

**Comment by mvelinder** [comment]
I'd say we rethink this on the whole, but right now you have it as a scrolling left/right panel. For large SVs that overlap hundreds/thousands of genes this becomes fairly unwieldy. At a minimum if you keep this design I'd suggest hiding/collapsing all the genes that have no association
![Image](https://github.com/user-attachments/assets/404954ec-50fd-4065-8466-8d669bfa3ebc)

**Comment by emersonlebleu** [comment]
> I'd say we rethink this on the whole, but right now you have it as a scrolling left/right panel. For large SVs that overlap hundreds/thousands of genes this becomes fairly unwieldy. At a minimum if you keep this design I'd suggest hiding/collapsing all the genes that have no association ![Image](https://github.com/user-attachments/assets/404954ec-50fd-4065-8466-8d669bfa3ebc)

Agreed on this the plan is to collapse genes that don't have any patient associations and then allow them to be expanded if needed.

**Comment by emersonlebleu** [comment]
Having this pane is a brand new experiment driven by removing this information from the other bar so there are lots of reworks/rethinks to do with it I think.

**Comment by AlistairNWard** [comment]
@mvelinder @emersonlebleu Is it worth having a meeting about this? I think the SV details card as a whole needs to be thought through properly and we need to make sure we're showing what really matters. I agree that this likely will be totally reworked, so it would be better to come up with a plan rather than tearing down what is there piecemeal!

**Comment by emersonlebleu** [comment]
> @mvelinder @emersonlebleu Is it worth having a meeting about this? I think the SV details card as a whole needs to be thought through properly and we need to make sure we're showing what really matters. I agree that this likely will be totally reworked, so it would be better to come up with a plan rather than tearing down what is there piecemeal!

I think it is worth it. The gene cards are there now because we needed to put the information somewhere when we took them out of the bar. I think at some point someone mentioned that it could go there. The phenotype and disease links & aggregation are one of the most useful features to Tom so I didn't want to push any update without that information available.

Other than that though I have some thoughts based on previous discussion about what could go there but it would be useful to solidify.

**Comment by mvelinder** [comment]
Yeah I think a meeting is probably warranted. And more efficient than issues ad infinitum. 

It'd also be good to take a step back to the original premise and design of the app as a whole. And what the critical things/information we want to be showing for analysis. It's easy to fall into a cycle of changing things that already exist - but we might not even need them to be there in the first place.

**Comment by emersonlebleu** [comment]
I think we have a first pass at this done. We are showing a small summary with duphold & svafotate annotations if they are available. We see only gene cards that have relevant information. What is relevant depends on what information the user has provided if we have phenotypes then we use that as the criteria for showing the gene on the variant details section.

We also see similar variants if there are any to speak of (from the svafotate bed) at .8 overlap.
